package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/honeycombio/libhoney-js.git"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": ">= 14.*"
```

Comment:
Let's test and confirm if engines in package.json is strict - does it ensure the new package isn’t updated (and thus breaking end users)? this helps us decide major vs minor bump. We should try with both npm and yarn.

Reply:
So far as I can tell, it will warn on the install but not stop it. ![Screen Shot 2022-09-13 at 6 01 15 PM](https://user-images.githubusercontent.com/15912063/190024256-2ec3ee6f-23e3-4d88-a7b6-6e7028669b97.png)

For what it's worth, in the example application the beeline still sends data AOK running node 12, thusfar

![Screen Shot 2022-09-13 at 5 58 36 PM](https://user-images.githubusercontent.com/15912063/190024297-941396df-ee12-4088-b7c9-0195a3a61bb2.png)